packages/sanity/src/ui/dialog/dialog.tsx

```diff
@@ -0,0 +1,53 @@
+import {Button, Flex, Dialog as UIDialog, DialogProps as UIDialogProps, Box} from '@sanity/ui'
+import React, {ComponentProps, forwardRef} from 'react'
+
+/** @internal */
+export interface DialogProps extends Omit<UIDialogProps, 'footer'> {
```

Comment:
To enforce visual consistency, props for Studio UI components should be as restrictive as possible. We should consider selectively _picking_  props rather than omitting, as currently it's possible to make per-instance overrides with custom `cardRadius`, `shadowRadius` etc.

Reply:
Updated! Thanks!